index.bs

```diff
@@ -179,13 +179,25 @@ a number of properties for itself:
 *   <dfn for="user agent" export>full version</dfn> (for example: "72.0.3245.12", "3.14159", or "297.70E04154A")
 *   <dfn for="user agent" export>platform brand</dfn> (for example: "Windows", "iOS", or "AmazingOS")
 *   <dfn for="user agent" export>platform version</dfn> (for example: "NT 6.0", "15", or "17G")
-*   <dfn for="user agent" export>platform architecture</dfn> (for example: "ARM64", or "ia32")
+*   <dfn for="user agent" export>platform architecture</dfn> (for example: "arm", or "x86")
 *   <dfn for="user agent" export>model</dfn> (for example: "", or "Pixel 2 XL")
 *   <dfn for="user agent" export>mobileness</dfn> (for example: ?0 or ?1)
 
 [=User agents=] SHOULD keep these strings short and to the point, but servers MUST accept arbitrary
 values for each, as they are all values constructed at the [=user agent=]'s whim.
 
+[=User agents=] MUST map higher-entropy [=platform architecture=] values to the following buckets:
+
+* x86 CPU architectures => "x86"
+* ARM CPU architectures => "arm"
+
+Other CPU architectures could be mapped into one of these values in case that makes sense, or be
+mapped to the empty string.
```

Comment:
For Apple platforms, we'd like the freedom to map even x86-family or arm-family architectures to the empty string, or to map an architecture to the "wrong" family when convenient.  It's not clear if this spec language is meant to allow that. I think this should be explicitly permitted, if this two-value field remains.

Reply:
This would be explicitly permitted by https://github.com/WICG/ua-client-hints/pull/90/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR205

Reply:
Sounds like it would. Might need some re-review once both PRs land.